### ISSUE TITLE
Motion sensors connected 

### ISSUE THREAD

**Issue by leo-b** [kukulich/home-assistant-jablotron100#48]

**Comment by kukulich** [comment]
I’m sorry I don’t understand.

**Comment by leo-b** [comment]
> I’m sorry I don’t understand.
Sorry. I accidently submitted the issue while still typing the description. :-)
I have now updated the text.

**Comment by kukulich** [comment]
Do the sensors have battery or not?

**Comment by leo-b** [comment]
No. They are old-style wired IR sensors with 12V power and NO/NC contacts. Power and signal lines are connected to this expansion card
https://www.jablotron.com/underwood/download/pdf/en/JA-116H.pdf

**Comment by kukulich** [comment]
@leo-b Can you enable the debug and sabotage the device 24? You can try to open it or something similar. It should log packets that may help.

**Comment by kukulich** [comment]
Should be resolved in https://github.com/kukulich/home-assistant-jablotron100/releases/tag/3.14.0

**Comment by leo-b** [comment]
> Should be resolved in https://github.com/kukulich/home-assistant-jablotron100/releases/tag/3.14.0

Confirmed.
Thanks!
